config/locales/sufia.en.yml

```diff
@@ -1,3 +1,34 @@
 en:
   sufia:
     deposit_agreement: "the OSU Libraries & Press Deposit Agreement"
+  simple_form:
+    labels:
+      generic_file:
+        tableOfContents: "Table of Contents"
+        bibliographicCitation: "Bibliographic Citation"
+        isReferencedBy: "Referenced By"
+        dc_type: "Resource Type"
+        isCitedBy: "Cited By"
+        isIdenticalTo: "Identical To"
+        isPartOf: "Part Of"
+        isVersionOf: "Version Of"
+        doi: "Digital Object Identifier"
+        hdl: "Handle Identifier"
+        orcid: "Open Researcher and Contributor ID"
+        rid: "Researcher ID"
+      metadata_help:
```

Comment:
@zhang4952 I tested these locales and I think there is an indentation issue with 'metadata_help'. The descriptions won't show up unless 'metadata_help' is moved one tab to the left so that it is inline with 'labels'. Then under 'metadata_help', there should be another 'generic_file' item as a child, and all the fields currently under 'metadata_help' would probably be moved under 'generic_file' (new child under 'metadata_help'). See https://github.com/projecthydra/sufia/blob/master/config/locales/sufia.en.yml